docs/element-templates/README.md

```diff
@@ -1,5 +1,3 @@
-> :warning: __The feature Element Templates may be subject to change in the future.__ As an example, this may lead to changes in the way you configure templates in JSON.
-
 # Element Templates
```

Comment:
Just to double-check, we still use the wording "Element Template" in technical documentation, right?

Reply:
Yes, for me the technical feature is still "element templates"

Reply:
I'm curious how this distinction will work out for our users. 👍